cloud/google/pods.go

```diff
@@ -34,7 +34,7 @@ import (
 
 var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.5"
 var controllerManagerImage = "gcr.io/k8s-cluster-api/controller-manager:0.0.6"
-var machineControllerImage = "gcr.io/k8s-cluster-api/gce-controller:0.0.13"
+var machineControllerImage = "gcr.io/k8s-cluster-api/gce-controller:0.0.14"
```

Comment:
we probably want to update the provider component yaml as well
https://github.com/kubernetes-sigs/cluster-api/blob/master/clusterctl/examples/google/provider-components.yaml.template#L47

Reply:
done.